app/models/government-field.js

```diff
@@ -0,0 +1,6 @@
+import Model, { attr, hasOne } from '@ember-data/model';
+
+export default class GovernmentFieldModel extends Model {
+  @attr('string') prefLabel;
+  @hasOne('government-domain') governmentDomain;
```

Comment:
`hasOne` bestaat niet in Ember. Enkel `belongsTo` en `hasMany`. Ook de inverse relaties specifieren.